ddclient.in

```diff
@@ -915,6 +915,17 @@ my %protocols = (
             'server'       => setv(T_FQDNP, 1, 0, 'domains.google.com', undef),
         },
     },
+    'henet' => {
```

Comment:
Should this be `he` for consistency with the `%builtinweb` entry?

Or should we add a `henet` alias in `%builtinweb`?

Reply:
You are right, `%builtinweb` and `%protocols` entries should indeed be consistent. I did consider `he`, `hurricane` and `henet`. 

I settled with `henet` for the initial review, but certainly open to change. Would you have a preference/recommendation?

Reply:
There's no reason it can't have a period in it, so how about `he.net`?  I think that name self-documents better than `he` or `henet`.  I can add `he.net` to `%builtinweb` for consistency.

Reply:
Sure, `he.net` sounds perfect!

Reply:
I pushed a fixup commit that changes it to `he.net`.